### ISSUE TITLE
Please remove sitaramc/* and gitolite/*

### ISSUE THREAD

**Issue by sitaramc** [tip4commit/tip4commit#173]
Please remove all mentions of both sitaramc/\* and gitolite/\* from your system.  I do not endorse what you are doing.

Despite the disclaimer you added very recently, I consider your website/service/whatever quite misleading.  Due to the lack of transparency (i.e., lack of "opt-in") and your apparent unwillingness to understand the concerns of projects owners, I have absolutelty no desire to have my name ("sitaramc") or my licensed trademark ("gitolite") be associated in any way shape or form with you, and I do not wish them to appear anywhere on your site except by way of resolving this complaint.

NOTE: GIT is a trademark of Software Freedom Conservancy and my use of "Gitolite" is under license.  I do not know if the SFC will care but I can certainly forward my concerns to them and ask their opinion (and intervention, if they agree with me) if this issue is not resolved.

Bottom line: you used my personal name and a licensed trademark name without permission.  Please don't think the GPL will get you out of this one; this has nothing to do with the code, and everything to do with people's and products' names and reputations.  By all means have someone fork gitolite **under a different name** and do whatever you want, but don't use "sitaramc" or "gitolite" anywhere.
